src/services/twilio.ts

```diff
@@ -45,7 +49,38 @@ export const TwilioClient = (): IPhoneProviderService => {
       return new UnknownPhoneProviderServiceError(err)
     }
 
-    logger.info({ to }, "sent text successfully")
+    return true
+  }
+
+  const validateVerify = async ({
+    to,
+    code,
+  }: {
+    to: PhoneNumber
+    code: PhoneCode
+  }): Promise<true | UnknownPhoneProviderServiceError | CodeInvalidError> => {
```

Comment:
please keep same signature of interface, also return should be only `Promise<true | PhoneProviderServiceError>`